modules/op-release-notes-1-1.adoc

```diff
@@ -0,0 +1,105 @@
+* The `kubeconfigwriter` now adds the `ClientKeyData` and the `ClientCertificateData` configurations in the Resource structure to enable replacement of the pipeline resource type cluster with the kubeconfig-creator Task.
+* The feature-flags config map names and the defaults config map name are now customizable.
+* Support for HostNetwork in the PodTemplate used by TaskRun is now added.
+* An Affinity Assistant is now introduced to support node affinity in TaskRuns that share workspace volume. By default, this is disabled on OpenShift Pipelines.
+* The PodTemplate has been updated to specify `imagePullSecrets` to identify secrets that the container runtime should use to authorize container image pulls when starting a Pod.
+* Support for emitting warning events from the TaskRun controller if the controller fails to update the TaskRun.
+* Standard or recommended k8s labels are now added to all resources to identify resources belonging to an application or component.
+* The Entrypoint process is now notified for signals and they are propagated using a dedicated PID Group of the Entrypoint process.
+* The PodTemplate is now allowed to be set on a Task level at runtime using `TaskRunSpecs`
+* Support for emitting Kubernetes events is added:
+** TaskRun lifecycle events such as the `taskrun started` and the `taskrun running` commands are now emitted.
+** The PipelineRun controller now emits an event every time that a Pipeline is started.
+* In addition to the default Kubernetes events, support for CloudEvents for TaskRuns is now added. The controller can be configured to send any TaskRun events, such as create, started, and failed as cloud events.
+* Support for using the `$context.<task|taskRun|pipeline|pipelineRun>.name` variable to reference the appropriate name when in PipelineRuns and TaskRuns.
+* Validation for PipelineRun parameters is now added to ensure that all the parameters required by the Pipeline are provided by the PipelineRun. This also allows PipelineRuns to provide extra parameters in addition to the required parameters.
+* You can now specify Tasks within a Pipeline that will always execute before the pipeline exits, either after finishing all tasks successfully or after a Task in the Pipeline failed, using the `finally` field in the Pipeline YAML file.
+* The `git-clone` ClusterTask is now added.
+
+
+=== Pipelines CLI
+
+* Support for embedded Trigger binding is now added to the `tkn evenlistener describe` command.
+* Support to recommend subcommands and make suggestions if an incorrect subcommand is used.
+* The `tkn task describe` command now auto selects the task if only one task is present in the Pipeline.
+* You can now start a Task using default parameter values by specifying the `--use-param-defaults` flag in the `tkn task start` command.
+* You can now specify a volumeClaimTemplate for PipelineRuns or TaskRuns using the `--workspace` option with the  `tkn pipeline` or  `tkn task start` commands.
```

Comment:
should we change it to  `tkn pipeline start` or  `tkn task start` 
or
 `tkn pipeline` or  `tkn task`  `start`

Reply:
Done